doc/modules/mod_pubsub.md

```diff
@@ -1,43 +1,59 @@
-    * `<<"virtual">>` - does not store nodes in database. This saves resources on systems when the number of nodes is large. If using the `<<“virtual”>>` nodetree, you can only enable those node plugins:
-    `[<<“flat”>>,<<“pep”>>]` or `[<<“flat”>>]`; any other plugins configuration will not work. Also, all nodes will have the defaut configuration, and this can not be changed.
+    * `<<"virtual">>` - does not store nodes in database.
+    This saves resources on systems when the number of nodes is large.
+    If using the `<<“virtual”>>` nodetree, you can only enable those node plugins:
+    `[<<“flat”>>,<<“pep”>>]` or `[<<“flat”>>]`; any other plugins configuration will not work.
+    Also, all nodes will have the defaut configuration, and this can not be changed.
     Using `<<“virtual”>>` nodetree requires to start from a clean database, it will not work if you used the default `<<“tree”>>` nodetree before.
     * `<<"dag">>"` - provides experimental support for [XEP-0248 (PubSub Collection Nodes)](http://xmpp.org/extensions/xep-0248.html).
-    In that case you should also add `<<“dag”>>` node plugin as default, for example: plugins: `[<<"dag">>,<<"flat">>,<<"hometree">>,<<"pep">>]`
-* `ignore_pep_from_offline` (boolean, default: `true`): specify whether or not we should get last published PEP items from users in our roster which are offline when we connect. Default option is `true` hence
-we will get only last items from online contacts.
-* `last_item_cache` (boolean, default `false`): specify whether or not pubsub should cache last items. Such a cashing might speed up pubsub's performance and allows to raise user connection rate but in price of memory usage.
-* `plugins` ([Plugin, ...], default: `[<<"flat">>]`): List of enabled pubsub plugins. They handle afﬁliations, subscriptions and items and also provide default node conﬁguration and features.
-PubSub clients can define which plugin to use when creating a node by adding `type=’plugin-name’` attribute to the create stanza element. If such an attribute is not specified, the default plugin will be the first on the plugin list.
-    * `<<"flat">>` -  No node hierarchy. It handles standard PubSub case
-    * `<<"hometree">>` - Use exact the same features as ﬂat plugin but additionally organise nodes in a tree. Basically it follows the scheme similar to ﬁlesystem's structure.
-    Every user can create nodes in its own home root: e.g `/home/user`. Each node can contain items and/or sub-nodes.
-    * `<<"pep">>` - implementation of [XEP-0060 (Personal Eventing Protocol)](http://xmpp.org/extensions/xep-0163.html). In this case, items are not persisted but kept in memory cache.
-    When the `pep` plugin enabled, user can have his own node (exposed on his bare jid) with a common namespace. Requires module `mod_caps` to be enabled.
+    In this case you should also add `<<“dag”>>` node plugin as default, for example: plugins: `[<<"dag">>,<<"flat">>,<<"hometree">>,<<"pep">>]`
```

Comment:
from the plugins description:
If client does not explicit specify the default plugin when creating the stanza then:

> (...) the default plugin will be the first on the plugin list. (...)

However, I must admin that p1 made pubsub configuration quite complicated ...